toolbox/include/CentroidalMomentum.h

```diff
@@ -2,21 +2,30 @@
 #define WBT_CENTROIDALMOMENTUM_H
 
 #include "WBIModelBlock.h"
+#include <iDynTree/Core/VectorFixSize.h>
 
 namespace wbt {
     class CentroidalMomentum;
 }
 
-class wbt::CentroidalMomentum : public wbt::WBIModelBlock {
+namespace iDynTree {
+    class Twist;
+    class Transform;
+    class VectorDynSize;
+    class SpatialMomentum;
+}
 
-    double *m_basePose;
-    double *m_centroidalMomentum;
+class wbt::CentroidalMomentum : public wbt::WBIModelBlock
+{
+    // Robot state
+    iDynTree::Twist*         m_baseVelocity;
+    iDynTree::Vector3*       m_gravity;
+    iDynTree::Transform*     m_world_T_base;
+    iDynTree::VectorDynSize* m_jointsVelocity;
+    iDynTree::VectorDynSize* m_jointsState;
```

Comment:
In mechanical systems, "state" is tipically used to indicate position ***and*** velocity, so I would call this member `m_jointsPosition` or something similar.

Reply:
I didn't like the previous word *configuration* and I switched to *State*. *Position* is even more clear :+1: